core/Migrations/Version13000Date20170718121200.php

```diff
@@ -454,6 +518,12 @@ public function changeSchema(IOutput $output, \Closure $schemaClosure, array $op
 			]);
 			$table->setPrimaryKey(['id']);
 			$table->addIndex(['class'], 'job_class_index');
+		} else {
+			$table = $schema->getTable('jobs');
+			$table->changeColumn('execution_duration', [
+				'notnull' => true,
```

Comment:
:see_no_evil: should be 
```suggestion
				'notnull' => false,
```

Otherwise we can't save when the duration was <1 second